src/libs/OptionsListUtils.js

```diff
@@ -78,6 +78,12 @@ Onyx.connect({
     },
 });
 
+let formattedPersonalDetails;
+Onyx.connect({
+    key: ONYXKEYS.PERSONAL_DETAILS,
+    callback: val => formattedPersonalDetails = val,
+});
```

Comment:
**thought:** we were avoiding this subscription by passing in the `personalDetails` whenever we use `getOptions()`. 

**question:** why do we need this now?

Reply:
This is used to get the avatars for group chats, an alternative solution is to pass the `personalDetails` using `withOnyx` to the components using `getReportIcons`.

Reply:
Update: I noticed `personalDetails` is already passed (using `withOnyx`) in a few components using `getReportIcons`, the only component that is not passing it is `ReportActionItemCreated`. I will remove the subscription and pass `personalDetails` to this component.